app/controllers/ratings_controller.rb

```diff
@@ -0,0 +1,20 @@
+class RatingsController < ApplicationController
+  def create
+    @bathroom = Bathroom.find(params[:bathroom_id])
+    if logged_in?
```

Comment:
Put this in a before_action to help keep the controller methods simple. These long if/else blocks get confusing quickly.